jenkins/scripts/perf/local/submit.py

```
@@ -357,22 +357,100 @@ def get_benchmark_config(config, benchmark_mode):
     try:
-        gres = (
-            subprocess.check_output(
-                ["sinfo", "-p", partition, "-h", "-o", "%G"],
-                stderr=subprocess.DEVNULL,
-                text=True,
-            )
-            .strip()
-            .split("\n")[0]
-            .strip()
+        out = subprocess.check_output(
+            ["sinfo", "-h", "-o", "%P"], stderr=subprocess.DEVNULL, text=True
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)